types/trace/trace.go

```diff
@@ -14,35 +14,36 @@ import (
-	ProcessorID          int          `json:"processorId"`
-	ProcessID            int          `json:"processId"`
-	CgroupID             uint         `json:"cgroupId"`
-	ThreadID             int          `json:"threadId"`
-	ParentProcessID      int          `json:"parentProcessId"`
-	HostProcessID        int          `json:"hostProcessId"`
-	HostThreadID         int          `json:"hostThreadId"`
-	HostParentProcessID  int          `json:"hostParentProcessId"`
-	UserID               int          `json:"userId"`
-	MountNS              int          `json:"mountNamespace"`
-	PIDNS                int          `json:"pidNamespace"`
-	ProcessName          string       `json:"processName"`
-	HostName             string       `json:"hostName"`
-	ContainerID          string       `json:"containerId"`
-	Container            Container    `json:"container,omitempty"`
-	Kubernetes           Kubernetes   `json:"kubernetes,omitempty"`
-	EventID              int          `json:"eventId,string"`
-	EventName            string       `json:"eventName"`
-	MatchedPolicies      uint64       `json:"-"` // omit bitmask of matched policies
-	MatchedPoliciesNames []string     `json:"matchedPolicies,omitempty"`
-	ArgsNum              int          `json:"argsNum"`
-	ReturnValue          int          `json:"returnValue"`
-	Syscall              string       `json:"syscall"`
-	StackAddresses       []uint64     `json:"stackAddresses"`
-	ContextFlags         ContextFlags `json:"contextFlags"`
-	Args                 []Argument   `json:"args"` // Arguments are ordered according their appearance in the original event
-	Metadata             *Metadata    `json:"metadata,omitempty"`
+	Timestamp               int          `json:"timestamp"`
+	ThreadStartTime         int          `json:"threadStartTime"`
+	ProcessorID             int          `json:"processorId"`
+	ProcessID               int          `json:"processId"`
+	CgroupID                uint         `json:"cgroupId"`
+	ThreadID                int          `json:"threadId"`
+	ParentProcessID         int          `json:"parentProcessId"`
+	HostProcessID           int          `json:"hostProcessId"`
+	HostThreadID            int          `json:"hostThreadId"`
+	HostParentProcessID     int          `json:"hostParentProcessId"`
+	UserID                  int          `json:"userId"`
+	MountNS                 int          `json:"mountNamespace"`
+	PIDNS                   int          `json:"pidNamespace"`
+	ProcessName             string       `json:"processName"`
+	HostName                string       `json:"hostName"`
+	ContainerID             string       `json:"containerId"`
+	Container               Container    `json:"container,omitempty"`
+	Kubernetes              Kubernetes   `json:"kubernetes,omitempty"`
+	EventID                 int          `json:"eventId,string"`
+	EventName               string       `json:"eventName"`
+	MatchedPolicies         uint64       `json:"-"`
+	MatchedPoliciesFiltered uint64       `json:"-"`
```

Comment:
```suggestion
	MatchedPoliciesKernel   uint64       `json:"-"`
	MatchedPoliciesUserland uint64       `json:"-"`
```

Perhaps? Suggestions?

Reply:
remove the land and I'm +1 on the rename
`MatchedPoliciesKernel`
`MatchedPoliciesUser`